castor.php

```diff
@@ -205,6 +205,14 @@ function lint_twig(): int
     return exit_code('bin/console lint:twig templates/');
 }
 
+#[AsTask(name: 'twig', namespace: 'stan', description: 'Lint Twig files with twigstan', aliases: ['lint-twigstan'])]
```

Comment:
This is not about linting. It's static analysis, same as PHPStan.

Reply:
Humm, I left the same “namespace” as the Symfony command `lint:twig`, but I realize this one is not a really a lint because it checks if the templates are valid and can be compiled, it's more like a static check as well.

Reply:
So indeed, it may be better to introduce another namespace `static` or `analyse`

Reply:
> it checks if the templates are valid and can be compiled, it's more like a static check as well.

It is **exactly** like PHPStan. It compiles the Twig templates to PHP, then optimizes the templates so that they are a bit easier to analyze, and then invokes PHPStan to analyze the code and report back its errors.

Reply:
The [Symfony definition](https://symfony.com/blog/new-in-symfony-7-2-translations-linter) does not make the difference between both:

> [Linters](https://en.wikipedia.org/wiki/Lint_(software)) are static code analysis tools used to flag errors like syntactic and stylistic issues. Symfony includes many linter commands to check YAML config files (lint:yaml), container services (lint:container), Twig templates (lint:twig) and even the syntax of XLIFF translation files (lint:xliff). 

So may it is better to keep lint for simplicity.